mkinitcpio

```diff
@@ -217,26 +217,21 @@ build_image() {
             ;;
     esac
 
-    cpio_opts=('-0' '-o' '-H' 'newc')
-    (( _optquiet )) && cpio_opts+=('--quiet')
-    if (( SOURCE_DATE_EPOCH )); then
-        cpio_bin="cpio"
-        cpio_opts+=('--reproducible')
-    else
-        cpio_bin="bsdcpio"
-    fi
+    cpio_opts=('--null' '-cf-' '--format=newc')
+    # TODO (( _optquiet )) && cpio_opts+=('--quiet')
```

Comment:
RFC. On my box, bsdtar is silent anyway.
I suppose we want this to appent -v if not present?